src/odemis/gui/cont/fastem_acq.py

```diff
@@ -251,12 +251,10 @@ def on_acquisition_done(self, future, num):
         try:
             da = future.result()
         except CancelledError:
-            self._reset_acquisition_gui()
+            # Just return because the error is handled in full_acquisition_done
```

Comment:
You can drop the special exception handling for `CancelledError`, as now it does the same as the `Exception`.